### PR TITLE
Use standard Jenkins POM in Winstone, Refactor the project structure

### PR DESCRIPTION
**Pull request by oleg-nenashev**
During the release of 4.1.1 I hit the issue with source uploads, which was failing the release due to the duplication of artifacts. So I decided to fix it and to align pom.xml with other Jenkins components.

This change introduces Jenkins parent POM 1.37 in the project and restructures it to be compatible with default project layout in the organization.

@reviewbybees @olamy

### CODE REVIEW

File: src/test/java/winstone/BoundedExecutorServiceDriver.java [hunk 1]
```diff
@@ -1,6 +1,8 @@
 package winstone;
 
 import junit.framework.TestCase;
+import winstone.BoundedExecutorService;
+import winstone.Launcher;
```
[the remark this answers is not in the document]
Comment (oleg-nenashev) [reply src/test/java/winstone/BoundedExecutorServiceDriver.java]: Likely it was a refactoring mess, will fix

**Comment by oleg-nenashev** [comment]
Since it's not urgent, I will try to integrate a new parent POM with FindBugs support.
I am tired of updating every core component to get it :)

**Comment by olamy** [comment]
Hi @oleg-nenashev 
I'd like to do some changes. So to avoid you or me some pain in merging maybe you should merge this one first?

**Comment by oleg-nenashev** [comment]
@olamy will do within a couple of hours

File: src/test/java/winstone/testCase/load/LoadTest.java [hunk 2]
```diff
@@ -27,6 +28,7 @@
  * @author <a href="mailto:rick_knowles@hotmail.com">Rick Knowles</a>
  * @version $Id: LoadTest.java,v 1.2 2006/02/28 07:32:49 rickknowles Exp $
  */
+@Ignore
```
[the remark this answers is not in the document]
Comment (oleg-nenashev) [reply src/test/java/winstone/testCase/load/LoadTest.java]: agreed

**Comment by oleg-nenashev** [comment]
comments have been addressed

File: src/test/resources/winstone/testCase/load/LocalStrings.properties [hunk 3]
```diff
@@ -1,5 +1,5 @@
 LoadTest.Usage=Winstone Command Line Load Tester\n\
-Usage: java winstone.testCase.load.LoadTest --url=<url> \
```
Comment (oleg-nenashev) [inline src/test/resources/winstone/testCase/load/LocalStrings.properties]: refactoring mess

**Comment by oleg-nenashev** [comment]
@olamy should I cut the release or wait for your patch to arrive?

**Pull request merged**